docs/guides/postcss.md

```diff
@@ -10,27 +10,26 @@ description: How to use PostCSS in your Snowpack project.
 
 [PostCSS](https://postcss.org/) is a popular CSS transpiler with support for [a huge ecosystem of plugins.](https://github.com/postcss/postcss#plugins)
 
-**To use PostCSS with Snowpack:** add the [@snowpack/plugin-postcss](https://www.npmjs.com/package/@snowpack/plugin-postcss) plugin to your project.
+**To use PostCSS with Snowpack:** Install [@snowpack/plugin-postcss](https://www.npmjs.com/package/@snowpack/plugin-postcss), [PostCSS](https://www.npmjs.com/package/postcss), and your PostCSS plugins, then add this plugin to your Snowpack config.
 
-```diff
-  // snowpack.config.mjs
-  export default {
-    plugins: [
-+    '@snowpack/plugin-postcss',
-    ],
-  };
+```js
+// snowpack.config.mjs
+export default {
+  plugins: ['@snowpack/plugin-postcss'],
+};
```

Comment:
I’d prefer the original `diff` that shows the addition

Reply:
Done. On the subject of `diff`, is the color swap intentional? Before additions were green and deletions red but now its reversed.

Reply:
Uh oh! Will look into that. Thanks for the heads up.